optuna/storages/rdb/storage.py

```diff
@@ -37,19 +37,46 @@
 class RDBStorage(BaseStorage):
     """Storage class for RDB backend.
 
-    This class is not supposed to be directly accessed by library users.
+    Note that library users can instantiate this class, but the attributes
+    provided by this class are not supposed to be directly accessed by them.
+
+    Example:
+
+        We create a :class:`~optuna.storages.RDBStorage` instance with
```

Comment:
It's trivial but I think it seems to be 'an RDBStorage' in terms of pronunciation.

```suggestion
        We create an :class:`~optuna.storages.RDBStorage` instance with
```

Reply:
Merged your suggestion. Thank you for the careful review!